### PR TITLE
Add text wrap to compute system status

### LINKED ISSUE #2721: Error message cut-off
### Dev Home version

0.1301.473.0

### Windows build number

_No response_

### Other software

_No response_

### Steps to reproduce the bug

My laptop failed to create a hyper-v image. But I do not know why. The error message is cut off

image

### Expected result

_No response_

### Actual result

I tried to create a hyper-v image.
it failed.

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by SohamDas2021**
## PR checklist
- [ ] Closes #2721
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: common/Environments/CustomControls/CardBody.xaml [hunk 1]
```diff
@@ -74,6 +74,7 @@
                         Visibility="{x:Bind ComputeSystemOperationStatus, Mode=OneWay, Converter={StaticResource EmptyStringToCollapsedConverter}}"
                         Text="{x:Bind CardState, Converter={StaticResource CardStateToLocalizedTextConverter}, Mode=OneWay}"
                         Style="{StaticResource CardBodyStateTextBlockStyle}"
+                        TextWrapping="Wrap"
```
Comment (krschau) [inline common/Environments/CustomControls/CardBody.xaml:77]: Do you want "Wrap" or "WrapWholeWords"?

Comment (SohamDas2021) [reply common/Environments/CustomControls/CardBody.xaml:77]: We are using Wrap in more places. Is there a convention?

Comment (dhoehna) [reply common/Environments/CustomControls/CardBody.xaml:77]: Whatever looks nice.

**Review by dhoehna — APPROVED** [review]

**Review by jsidewhite — APPROVED** [review]
:shipit:

File: common/Environments/CustomControls/CardBody.xaml (hunk 1 above)
Comment (krschau) [reply common/Environments/CustomControls/CardBody.xaml]: It depends on whether it makes sense to let a word be cut off in the middle. I would argue that usually, you don't want that. https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.textwrapping

**Review by krschau — APPROVED** [review]

**Pull request merged**